repo2docker/utils.py

```diff
@@ -68,24 +68,17 @@ def chdir(path):
         os.chdir(old_dir)
 
 
-def validate_and_generate_port_mapping(port_mapping):
+def validate_and_generate_port_mapping(port_mappings):
     """
-    Validate the port mapping list and return a list of validated tuples.
-
-    Each entry in the passed port mapping list will be converted to a
```

Comment:
Is there a reason you deleted this docstring?

Reply:
I tried to update it as parts weren't describing what actually happened anymore.

Not sure what the best way to describe the returned dictionary is, update the docstring to link to the docker-py method that consumes this port mapping.